modules/nextflow/src/main/groovy/nextflow/script/parser/v2/ErrorListener.groovy

```diff
@@ -90,7 +90,7 @@ class StandardErrorListener implements ErrorListener {
 
     @Override
     void onError(SyntaxException error, String filename, SourceUnit source) {
-        term.bold().a(filename).reset()
+        term.bold().a("error: ").a(filename).reset()
```

Comment:
Let me know if you like how this looks and I'll merge. I wonder if it would be better to put the error/warning label after the location

Reply:
Haven't checked, but yeah I was thinking after the file name - like a prefix to the error message:

```
Error: Something bad happened.
```

Maybe with some colour too? Need to look at it for that though.

Reply:
Updated. Let me know how it looks